### DISCUSSION TITLE
Previews problem while running hugo server [Solved]

### DISCUSSION THREAD

**Discussion by gitaeron** [Lednerb/bilberry-hugo-theme#449] [General]
I know the title is weird, but I don't know how to describe.... 

## Multiple pictures contained

You can read this in [My Blog](https://snoer.cyou/tech/hugo-server/) to have a better reading experience. &emsp;  : )

---

# What happened

There's some operation caused 404 page while running `hugo server` .

Once you changed `categories` and `tags` during running `hugo server` , browser'll jumped to 404 page after you click the changed `categories` and `tags`.

---

# Process description

For example:

1. This article's `categories` and `tags` is like this

- categories: ['tech']
- tags: ['hugo', 'issue', 'experience']

![1](https://raw.githubusercontent.com/gitaeron/gitimg/main/tech/hugo-server/1.png)

<br/>

2. After running `hugo server` , you can simply click `categories` or `tags` jump to corresponding page

![2](https://raw.githubusercontent.com/gitaeron/gitimg/main/tech/hugo-server/2.png)

<br/>

3. But after you changed `categories` or `tags` in editor
   
![3](https://raw.githubusercontent.com/gitaeron/gitimg/main/tech/hugo-server/3.png)

<br/>

![4](https://raw.githubusercontent.com/gitaeron/gitimg/main/tech/hugo-server/4.png)

<br/>

4. Turns out with 404

![5](https://raw.githubusercontent.com/gitaeron/gitimg/main/tech/hugo-server/5.png)

<br/>

5. When you're running `hugo server` , this situation includes but is not limited to `categories` or `tags` or `author` etc 

![6](https://raw.githubusercontent.com/gitaeron/gitimg/main/tech/hugo-server/6.png)

<br/>
   
6. It also works for adding new one after running `hugo server`

![7](https://raw.githubusercontent.com/gitaeron/gitimg/main/tech/hugo-server/7.png)

---

# Solution

**Just simply `Ctrl` + `C`**

**Then running `hugo server`**

![8](https://raw.githubusercontent.com/gitaeron/gitimg/main/tech/hugo-server/8.png)

---

# Conclusion

I don't know why, but it is as it is....

I‘m not even a front-end engineer anyway.... &emsp; :(

Is this article like a kind of simply tips?

**Comment by igor-baiborodine** [comment]
@gitaeron, or you can simpy use the `--disableFastRender` flag to start the Hugo server: 

![image](https://user-images.githubusercontent.com/1027701/204532788-48d643a4-adf1-4f18-bb5c-23b0b7d90eee.png)
